src/main.cpp

```diff
@@ -552,6 +552,16 @@ bool IsMay152018Enabled(const Consensus::Params &consensusparams, const CBlockIn
     return pindexPrev->IsforkActiveOnNextBlock(miningForkTime.value);
 }
 
+bool IsMay152018Next(const Consensus::Params &consensusparams, const CBlockIndex *pindexPrev)
+{
```

Comment:
Why not using `pindexPrev->forkAtNextBlock` rather than defining another wrapper?

Reply:
I originally wrote it direct like that, but then I thought that since we have 1 wrapper I should keep to that for consistency.  This way, post fork we can just search for "May152018" to get all the code points that need to be converted to a specific block height.

Reply:
fair enough. 

the only thing is that we already check for `pindexPrev` being a `nullptr` in `forkAtNextBlock` so we are doing it twice.

Reply:
yeah... checking arg validity in short helper functions is inefficient but realistically with today's CPUs its no difference unless inside an inner loop